src/pages/Swap/components/TransactionDetails.tsx

```diff
@@ -142,19 +146,26 @@ export function TransactionDetails({
                 type="number"
                 InputProps={{
                   inputProps: {
-                    min: 0,
+                    min: MIN_SLIPPAGE,
                     max: 100,
+                    step: 0.1,
                   },
                 }}
                 onChange={(e) => {
                   const value = e.target.value;
-                  const isValid = isSlippageValid(value);
-                  if (!isValid) {
-                    return;
+                  if (isSlippageValid(value)) {
```

Comment:
I think this change handler makes the UX a bit jarring. When 0.1 is reached, there is no way to delete and enter a number. Also when I enter a `0.` to enter let's say `0.5` it jumps to `0.1` and it's not very user friendly to get it to the desired number.

https://github.com/user-attachments/assets/93a743fa-54fe-4195-a161-671b895daf0f